public/js/train.js

```diff
@@ -113,6 +113,12 @@ $(document).ready(function() {
   }
 
   $('button[type=reset]').click(function() {
+    if ($(event.target).hasClass('reset--classifier')) {
+      if (!confirm("Are you sure you want to delete the classifier?")) {
+        $(event.target).blur();
+        return;
```

Comment:
I think you'd want to call `event.preventDefault()` here, that's the normal way of canceling a click. (Assuming we do use this.)

Does `$(event.target).blur();` cancel the click?